unitstatushandler/softwaremanager.go

```diff
@@ -1520,17 +1520,17 @@ func (manager *softwareManager) updateNodes() (nodesErr error) {
 	return nodesErr
 }
 
-func (manager *softwareManager) updateUnitConfig() (unitConfigErr error) {
+func (manager *softwareManager) updateUnitConfig() (err error) {
```

Comment:
Don't think changes in softwaremanger required

Reply:
fixed